### DISCUSSION TITLE
How to install multi-version packages and create symbol link in custom directory

### DISCUSSION THREAD

**Discussion by Whirlwind** [Homebrew/discussions#4203] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 3.6.21-89-ged79381-dirty
ORIGIN: https://github.com/Homebrew/brew
HEAD: ed793811553eb6fca3dd602931e37b7187f86a68
Last commit: 19 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: bbad80d836d9ead418e0ec858b645203f11c29e1
Core tap last commit: 2 weeks ago
Core tap branch: master
Core tap JSON: 13 Feb 10:20 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.39.1 => /opt/homebrew/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2-arm64
CLT: N/A
Xcode: 14.2
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready
```

### Description of issue

I want to install multi-version packages in `/opt/homebrew/Cellar`, then my project will use the specific version by the symbol link binary.

eg: A project requires `git 2.38.1`, and B project requires `git 2.37.1`.

I hope install the git in '/opt/homebrew/Cellar/git'.
there are two symbol links:
```
A/tools/git -> /opt/homebrew/Cellar/git/2.38.1
B/tools/git -> /opt/homebrew/Cellar/git/2.37.1
```

In my project , I will use `./tools/git/bin/git` to instead of the `git`. 

I try to install the multi versions by `brew extract` and `brew pin`. But I could not set the symbol link to my project from `/opt/homebrew/bin`.

**Comment by p-linnane** [comment]
This is not supported. You might try using something like [asdf](https://asdf-vm.com).
